compute-labs/aws/aws-ec2-with-ubuntu-and-nginx-web-server.md

```diff
@@ -52,7 +52,7 @@ In this exercise, you will provision an EC2 instance that runs Ubuntu Linux oper
 11. Click on the ![Next: Configure Security Group button](media/aws-next-configure-security-group-button.png) button to go to the next step.
 12. On the *Configure Security Group* screen, change the following:
   - *Security group name* → `[initials]-awsec2nginxweblab-sg`, where `[initials]` are your first, middle, and last name initials
-  - *Security group description* → `[your_full_name]security group for my first EC2 compute lab on AWS`
+  - *Security group description* → `[your_full_name]'s security group for my first EC2 compute lab on AWS`
```

Comment:
I thought the issue is that we cannot use apostrophes `'` in the description. You should just put space between `[your_full_name]` and `security` and not add back the apostrophe.

Reply:
You're right, I think there was some miscommunication. I thought it should have been removed but reverted the changes. My apologies.